### PR TITLE
Add 10kb limit (closes #48)

### LINKED ISSUE #48: 10 cookies per cookie jar limit
Hi everyone!

In the CHIPS explainer, it mentions that 

> User agents must limit third-party domains to just one or some small number of cookies per-partition

https://github.com/privacycg/CHIPS#limit-the-number-of-cookies-a-third-party-can-use-in-a-single-partition

The Memory Impact section also mentions research on choosing `10` as that cookie limit per partition (where partitions are based on top-level third-party domains):

https://github.com/privacycg/CHIPS#memory-impact

This is in comparison to today's behavior of imposing a cookie limit per-page for all first- and third-party domains (e.g. of 180+).

We're trying to understand whether a per-partition limit (of 10 or less) may have an impact in some cases, i.e. potentially causing existing scenarios to break.

One scenario we think this could impact is for services like CDNs that offer TLDs that host their customer's content.  For example, there are domains like `*.akamaihd.net` that many of our Akamai CDN customers may use.  Each customer has their own subdomain of `akamaihd.net`, such as `customer1.akamaihd.net`.

(Some Akamai CDN customers CNAME their own domains pointing to `customer1.akamaihd.net`, but others just use their given `customer1.akamaihd.net` directly)

Those Akamai CDN customers using `*.akamaihd.net` may be providing third-party resources for yet _other_ websites (Akamai customers or not).  So today there are websites with multiple `*.akamaihd.net` resources from different vendors all on the same page.

Will this be an issue if there are only 10 cookies limited to each partitioned jar?  We're honestly not sure.  We have a lot of customers, some of which deliver 3rd-party content, and they could each have their own several cookies they set.  If multiple of our customers's contents are embedded on another website, it's feasible that they could start to trip over the 10 cookie-per-partition limit.

We think there may be _some_ cases where 10 is not enough in practice, because even combinations of our own security products may set 5+ cookies.  A 15 cookie limit seems less likely to cause a problem than 10, but we're not sure what a reasonable number would be.

(Note we're also in the process of getting `akamaihd.net` on the [Public Suffix List](https://www.akamai.com/blog/developers/adding-akamai-shared-domains-to-the-public-suffix-list) which we understand would ensure subdomains to `akamaihd.net` get their own partitions.  That may take a while to be completed though.  In the meantime, we figured we'd open this issue as there may be others with similar concerns).

### PR DESCRIPTION
**Pull request by johannhof**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -439,6 +439,8 @@ A third-party domain's cookie jar should have a much lower per-partition size li
 User agents must limit third-party domains to just one or some small number of cookies per-partition.
 The number of cookies in a single partition per third-party is scoped by domain so that a third-party could not circumvent this limit by registering new subdomains.
 
+Based on [feedback on this proposal](https://github.com/privacycg/CHIPS/issues/48) that both a lower limit of cookies per-partition as well as a lower maximum size of individual partitioned cookies will pose problems for developers, we suggest a limit based on the total amount of memory used up by cookies of a partitioned site. This limit could be 10kb.
```
[the remark this answers is not in the document]
Comment (johannhof) [reply README.md]: Thanks!

File: README.md [hunk 2]
```diff
@@ -483,7 +485,7 @@ This would ensure that cross-site cookies are hostname bound and only sent over
 
 ### Memory impact
 
-Based on Chrome data, we estimate limiting a domain to 10 cookies per-partition will satisfy ~99% of existing cross-site cookie use cases on the web today.
+Based on Chrome data, we estimate that 10 cookies per-partition will satisfy ~99% of existing cross-site cookie use cases on the web today.
```
Comment (DCtheTall) [inline README.md:488]: Kind of unrelated, but we actually have browser metrics for this that show this was an underestimate on the number of cookies domains use. Wondering if this section needs a refactor.

**Review by DCtheTall — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (johannhof) [reply README.md:488]: Ok, do you want to do that at some point? I don't really have a good overview of those metrics.

**Comment by johannhof** [comment]
I don't seem to have enough permissions on this repo to merge this PR in @DCtheTall.

**Pull request merged**